### ISSUE TITLE
[Fixed in GH-6052] Safari & Firefox: Missing Visual Focus on Checkout: Choose Payment Method

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4326]
Even though the buttons **Add New Payment** and **Back** have visual focus this is not perceivable enough for accessibility purposes, it should be treated as missing visual focus.

Users with disabilities should be able to select or change payment methods using the keyboard, currently the option **Use this payment** is not keyboard accessible, therefore not receiving focus, thus no visual focus.

Buttons also missing visual focus

![Screen Shot 2019-08-29 at 7 21 32 PM](https://user-images.githubusercontent.com/15113219/63983030-f078df00-ca92-11e9-88db-4c260f02493e.png)

**Comment by Zeyber** [comment]
To be closed by #4993

**Comment by KateChuen** [comment]
Chrome OK

Missing keyboard accessibility and visual focus on:
- Set as default checkbox on Safari
- Back & Continue buttons on Safari
- Add New Payment on Safari
- Use this payment button on Firefox

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-30 at 3 43 14 PM](https://user-images.githubusercontent.com/15113219/73488626-45d55700-4377-11ea-8834-79314b9e8471.png)
![Screen Shot 2020-01-30 at 3 41 36 PM](https://user-images.githubusercontent.com/15113219/73488628-45d55700-4377-11ea-91d2-6b2c19834d6f.png)
![Screen Shot 2020-01-30 at 3 40 46 PM](https://user-images.githubusercontent.com/15113219/73488629-466ded80-4377-11ea-8997-c9c29be1db93.png)
![Screen Shot 2020-01-30 at 3 41 23 PM](https://user-images.githubusercontent.com/15113219/73488630-466ded80-4377-11ea-9ee0-77f15e333162.png)

**Comment by KateChuen** [comment]
Tested on dev3.
chrome ✅
firefox ✅
safari ✅
